edx_api/integration_test.py

```diff
@@ -395,3 +395,21 @@ def test_user_info_timeout():
         get.side_effect = mocked_timeout
         with pytest.raises(Timeout):
             user_info.get_user_info()
+
+
+@require_integration_settings
+def test_enrollments_list():
+    """
+    Enrolls the user in a course and then pulls down the enrollments for the user.
+    This assumes that the course in the edX instance is available for enrollment.
+    """
+    api = EdxApi({'access_token': ACCESS_TOKEN}, base_url=BASE_URL)
+    enrollments = api.enrollments.get_enrollments()
+    print(enrollments)
```

Comment:
We probably shouldn't print out extraneous text since the test output is pretty verbose already.

Reply:
Ah, yes. I was testing out the pagination. Sorry about that.